sdk/face/ai-vision-face-rest/samples-dev/livenessSessionWithVerifyWithVerifyImage.ts

```diff
@@ -0,0 +1,122 @@
+import { randomUUID } from 'crypto';
+
+import { AzureKeyCredential } from '@azure/core-auth';
+
+import createFaceClient, {
+    isUnexpected,
+} from '@azure-rest/ai-vision-face';
+import { readFileSync } from 'fs';
+
+/**
+ * This sample demonstrates how to create a liveness detection session.
+ *
+ * @summary creates a liveness detection session
+ */
+
+const pressAnyKeyToContinue = () => new Promise<void>(resolve => {
+    const { stdin } = process;
+    const { isRaw } = stdin;
+    stdin.setRawMode(true);
+    stdin.resume();
+    stdin.once('data', () => {
+        stdin.pause();
+        stdin.setRawMode(isRaw);
+        resolve();
+    });
+});
+
+const waitForLivenessRequest = async () => {
+    // Wait for request from client device.
+};
+
+const sendTokenToClientDevices = async (token: string) => {
+    // Send the token to client devices.
+};
+
+const waitForLivenessSessionComplete = async () => {
+    console.log('Please refer to https://learn.microsoft.com/en-us/azure/ai-services/computer-vision/tutorials/liveness and use the mobile client SDK to perform liveness detection on your mobile application.');
+    console.log('Press any key to continue when you complete these steps to run sample to get session results...');
+    await pressAnyKeyToContinue();
+};
+
+async function main() {
+    // This sample follows the documentation: https://learn.microsoft.com/en-us/azure/ai-services/computer-vision/tutorials/liveness
+    // We will follow the steps in https://learn.microsoft.com/en-us/azure/ai-services/computer-vision/tutorials/liveness#orchestrate-the-liveness-solution to demo the sample code in app server.
```

Comment:
Link should change to with verify section
https://learn.microsoft.com/en-us/azure/ai-services/computer-vision/tutorials/liveness#perform-liveness-detection-with-face-verification